config/global.php

```diff
@@ -72,4 +72,16 @@
                 'tld' => false,
             ))));
     },
+
+    'tracker.request.processors' => array(),
```

Comment:
Because it could be simpler. Every step that one has to do and could be avoided is too much. Especially here where one also has to touch another file (possible it will be created at some point but still it could be avoided maybe) Ideally plugin developers should not have to know about DI unless they really wanna replace things. Not every plugin developer will be familiar with DI, etc. Also one needs to read more documentation and it can be frustrating to find this config entry in case there is any request processor but it doesn't work because of missing config.